db/migrate/20190315055139_create_volunteers.rb

```diff
@@ -14,7 +14,20 @@ def change
       t.string :state
       t.string :employer
       t.string :position
-      t.string :availability
+      t.string :avail_sun_am
```

Comment:
This file should not be changed.  It should be restored as it was.  Tilneil, you have subsequent migration files that make the required changes, so you don't need to edit this file.